website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -224,11 +252,12 @@ components: sinks: elasticsearch: {
 				default: "_doc"
 			}
 		}
-		endpoint: {
```

Comment:
I think we should leave the option in here, just mark it as deprecated, per https://github.com/vectordotdev/vector/blob/master/docs/DEPRECATION.md.

Reply:
That requires adding `deprecation` field to schema of options. Should that change be a separate PR to be merged before this one?

Reply:
Ideally yes but I'm ok with just adding "deprecated" to the description for now.